std/ndn/fch/fch.go

```diff
@@ -0,0 +1,147 @@
+// Package fch provides a simple NDN-FCH client.
+// https://github.com/11th-ndn-hackathon/ndn-fch
+package fch
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/gorilla/schema"
+)
+
+var encoder = schema.NewEncoder()
```

Comment:
Personally, I don't like using a global variable.

Reply:
gorilla/schema recommends defining `encoder` and `decoder` as package globals.
This allows the instance to cache metadata internally, which improves performance.